spec/views/user_index_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+require 'capybara/rspec'
+
+RSpec.feature 'UsersIndices', type: :feature do
+  let(:user) do
+    User.create(
+      name: 'John Doe',
+      email: 'john.doe+2@example.com',
+      bio: "Hello, I'm John!",
+      posts_counter: 0
+    ).tap do |u|
+      u.photo.attach(io: File.open(Rails.root.join('app', 'assets', 'images', 'profile.jpg')), filename: 'profile.jpg')
+    end
+  end
+
+  def image_exists?(url)
+    response = Net::HTTP.get_response(URI.parse(url))
+    response.code == '200'
+  rescue StandardError
+    false
+  end
+
+  it 'displays the user name and profile image on the index page' do
```

Comment:
Good job having the set-up for the user index page, however you still have a couple of tests to write such as

- Tests that assert you can see the username of all other users.
- Tests that assert you can see the number of posts each user has written.
- Tests that redirect to the user's show page when you click on a user.

consider adding the above mentioned test in order to get the specs complete for this page